### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#1] [Announcements]
# about/

关于一个在时间轴上行走的人类。

https://blog.gztime.cc/about/

**Comment by GZTimeWalker** [comment]
第一个前排一定是自己的啦！😋😋😋

**Comment by miaotony** [comment]
好耶！
GZTime 新年快乐~

**Comment by TonyCrane** [comment]
好耶！

**Comment by YuyangLee** [comment]
G宝放心飞，妈妈永相随😘😘

**Comment by Little-Bla-ck** [comment]
性格很可爱的男孩子哦，技术力很强，加油~（等我出来去抱你大腿）

**Comment by Reverier-Xu** [comment]
是GZTime！抓！

**Comment by Brannua** [comment]
cool~

**Comment by GoodBoyboy666** [comment]
Travelling . . .

**Comment by GrakePch** [comment]
(～￣▽￣)～

**Comment by MinjieY** [comment]
超喜欢这个站的设计，想问下是博主自己设计的吗~

**Reply by GZTimeWalker** [reply]
基于 Butterfly 主题，自己做了一些小的样式修改，可以从页脚看到相关链接。

**Comment by hxy860721** [comment]
寻求合作  陕西汉中  40323484@qq.com

**Comment by luoye-cn** [comment]
hi 博主你好 我是 DeepSeek HR，看到你的个人网站和 GitHub 特来联系，不知道博主现在是什么状态，有没有看外部机会，DeepSeek 目前有大量岗位在招聘，如果有意向，欢迎添加我的微信：yxy2489，我们可以线上沟通下岗位信息～ 祝一切顺利！
